packages/xsnap/package.json

```diff
@@ -12,46 +12,40 @@
   },
   "scripts": {
     "repl": "node src/xsrepl.js",
-    "build:bundle": "rollup --config rollup.config.js",
     "build:bin": "node src/build.js",
-    "build:env": "node src/build.js --show-env > dist/build-env",
-    "build:from-env": "{ cat dist/build-env; echo node src/build.js; } | xargs env",
-    "build": "yarn build:bin && yarn build:bundle && yarn build:env",
+    "build:env": "node src/build.js --show-env > build-env",
+    "build:from-env": "{ cat build-env; echo node src/build.js; } | xargs env",
+    "build": "yarn build:bin && yarn build:env",
```

Comment:
The move from below the `dist` folder is causing a new dirty `build-env` file when building xsnap. Was that meant to be the case? I also don't know how it passed git dirty checks in CI